releases.yaml

```
@@ -1,4 +1,177 @@
+            rapidscada: 6.1.4-1
+            sensor-tools-scada-client: '1.1'
+            serial-tool: 1.2.0
+            sigrok-cli: 0.7.2-1
+            ss-dev: 2.0-1.46.2-2+wb1
+            tailscale: 1.62.1
+            task-wb-base-system: 1.18.4
+            task-wb-common-pkgs: 1.18.4
+            telegraf-wb-cloud-agent: 1.29.1
+            u-boot-tools-wb: 2:2021.10+wb1.7.1
+            wb-ble-tesliot: 1.1.1
+            wb-cc2652p-flasher: 1.0.0
+            wb-cloud-agent: 1.3.3
+            wb-configs: 3.22.1
+            wb-daemon-watchdogs: '1.1'
+            wb-demo-kit-configs: 1.7.0
+            wb-device-manager: 1.7.0
+            wb-diag-collect: 1.8.6
+            wb-dt-overlays: 1.6.0+wb1
+            wb-ec-firmware: 1.2.1
+            wb-essential: 1.18.4
+            wb-firmware-realtek: 1.0.3
+            wb-homa-adc: 2.6.5
+            wb-homa-gpio: 2.13.1
+            wb-homa-ism-radio: 1.17.3
+            wb-homa-rfsniffer: 1.0.9
+            wb-homa-w1: 2.2.10
+            wb-hwconf-manager: 1.58.8
+            wb-knxd-config: 1.1.2
+            wb-mb-explorer: 1.2.8
+            wb-mcu-fw-flasher: 1.3.2
+            wb-mcu-fw-updater: 1.10.11
+            wb-modbus-ext-scanner: 1.2.3
+            wb-mqtt-adc: 2.6.5
+            wb-mqtt-apcsnmp: '0.2'
+            wb-mqtt-bmp085: '1.2'
+            wb-mqtt-co2mon: 1.1.1
+            wb-mqtt-confed: 1.14.6
+            wb-mqtt-dac: 1.2.4
+            wb-mqtt-db: 2.8.13
+            wb-mqtt-db-cli: 1.4.5
+            wb-mqtt-gpio: 2.13.1
+            wb-mqtt-homeui: 2.82.1
+            wb-mqtt-iec104: 1.1.5
+            wb-mqtt-knx: 1.12.7
+            wb-mqtt-logs: 1.4.4
+            wb-mqtt-mbgate: 1.6.0
+            wb-mqtt-metrics: 0.3.3
+            wb-mqtt-opcua: 1.1.2
+            wb-mqtt-rfblinds: 1.0.2
+            wb-mqtt-serial: 2.116.1
+            wb-mqtt-smartbus: '1.2'
+            wb-mqtt-smartweb: 1.4.4
+            wb-mqtt-snmp: 1.2.7
+            wb-mqtt-urri: 1.2.1
+            wb-mqtt-w1: 2.2.10
+            wb-mqtt-zabbix: '0.2'
+            wb-nm-helper: 1.33.1
+            wb-rules: 2.20.1
+            wb-rules-system: 1.11.0
+            wb-suite: 1.18.4
+            wb-test-suite: '1.20'
+            wb-test-suite-deps: 1.12.0
+            wb-test-suite-dummy: 1.18.4
+            wb-update-manager: 1.3.2
+            wb-update-notifier: 0.1.0
+            wb-utils: 4.20.5
+            wb-zigbee2mqtt: 1.3.4
+            webif: '1.5'
+            z-way-server: 4.1.1-lws16
+            z-way4wb: '1.4'
+            zbw: '1.2'
+            zigbee2mqtt: 1.35.3-wb101
+            zigbee2mqtt-1.18.1: 1.18.1-wb102
+
+        wb6/bullseye:
+            <<: *packages-wb-2404
+
+            linux-headers-wb6: 5.10.35-wb164
+            linux-image-wb6: 5.10.35-wb164
+            u-boot-wb6: 2:2021.10+wb1.7.1
+            wb-bootlet-wb6x: 5.10.35-wb164-fs1.2.2-deb11-202311241137
+
+        wb7/bullseye: &packages-wb-2404-wb7-bullseye
+
```

Это нужно только для кастомных релизов на базе данного релиза
```suggestion
        wb7/bullseye:
```